### PR TITLE
♻️Migrate Stories 62..65 to `args`

### PR DESCRIPTION
**Pull request by rbeckthomas**
Partial for #35923.

This PR also removes the allowlist for the usage of `@storybook/addon-knobs` and references to it.

### CODE REVIEW

**Comment by samouri** [comment]
@rebeccanthomas: is this ready for review?

**Comment by rebeccanthomas** [comment]
> 

Ah, Not yet. This is blocked pending the merges of the other story migration. Sorry about that!

**Comment by rebeccanthomas** [comment]
This PR is blocked pending merge of #37697, #37727 and updates 45..49 (soundcloud, lightbox, and timeago).

File: src/preact/storybook/Wrappers.js [hunk 1]
```diff
@@ -1,62 +1,75 @@
...
-      style={style}
-      wrapperClassName={wrapperClassName}
-      wrapperStyle={wrapperStyle}
-      contentClassName={contentClassName}
-      contentStyle={contentStyle}
-      size={size}
-      layout={layout}
-      paint={paint}
       aria-label={ariaLabel}
+      {...args}
```
Comment (dmanek) [inline src/preact/storybook/Wrappers.js:42]: `args` contains some of the same properties that are passed in. Perhaps you want to do:
```
<ContainWrapper {...args} as={asProp} class={className} ...
```

File: src/preact/storybook/Wrappers.js [hunk 2]
```diff
@@ -1,62 +1,75 @@
...
+  wrapperClassName: '',
+};
+
+wrapper.argTypes = {
+  style: {
+    name: 'style',
+    defaultValue: {border: '1px solid'},
+    control: {type: 'object'},
+  },
+  wrapperStyle: {
```
Comment (kvchari) [inline src/preact/storybook/Wrappers.js:30]: does this need the default value of `{padding: 4}`?

File: src/preact/storybook/Wrappers.js [hunk 3]
```diff
@@ -1,62 +1,75 @@
...
+  paint: true,
+};
+
+containWrapper.argTypes = {
+  style: {
+    name: 'style',
+    defaultValue: {border: '1px solid', width: 200, height: 50},
+    control: {type: 'object'},
+  },
+  wrapperStyle: {
```
Comment (kvchari) [inline src/preact/storybook/Wrappers.js:66]: does this need the default value of `{padding: 4}`?

**Review by kvchari — APPROVED** [review]

**Review by dmanek — APPROVED** [review]

File: src/preact/storybook/Wrappers.js (hunk 3 above)
Comment (rbeckthomas) [reply src/preact/storybook/Wrappers.js:66]: ah yes! thank you :)

File: src/preact/storybook/Context.js [hunk 4]
```diff
@@ -1,36 +1,37 @@
...
   title: '0/Context',
-  decorators: [withKnobs],
 };
 
 const IMG_SRC =
   "data:image/svg+xml;charset=utf-8,%3Csvg xmlns='http://www.w3.org/2000/svg' width='24px' height='24px' viewBox='0 0 24 24' fill='%23000000'%3E%3Cpath d='M0 0h24v24H0z' fill='none'/%3E%3Cpath d='M9 16.17L4.83 12l-1.42 1.41L9 19 21 7l-1.41-1.41z'/%3E%3C/svg%3E";
 
-const LOADING_OPTIONS = ['auto', 'lazy', 'eager', 'unload'];
-
-export const _default = () => {
```
Comment (samouri) [inline src/preact/storybook/Context.js:19]: FMI: why is this named _default? Does that do something?

**Review by samouri — APPROVED** [review]

File: src/preact/storybook/Context.js (hunk 4 above)
Comment (rbeckthomas) [reply src/preact/storybook/Context.js:19]: As a convention we name the first story `_default` to make it clear its one that gets default selected when you navigate to the story. It doesnt do anything special in the storybook its more just for us 

the `default` on line 9/6 That one does some storybook internal magic and is important to be named that way

**Pull request merged**